odtp/dashboard/utils/middleware.py

```diff
@@ -0,0 +1,61 @@
+import jwt, logging
+from jwt import PyJWKClient
+from fastapi import Request, status, HTTPException, Cookie, Response
+from starlette.middleware.base import BaseHTTPMiddleware
+from nicegui import Client,app
+from odtp.dashboard.utils.storage import storage_update_user, get_from_storage
```

Comment:
todo: The function `get_from_storage` is not used any more, therefore it should not be imported